src/components/util/overlays/popover/usePopover.ts

```diff
@@ -30,7 +30,7 @@ export const usePopover = <E extends HTMLElement>(
   controller: PopoverController,
   options: undefined | UsePopoverOptions = {},
 ): PopoverProps<E> => {
-  const { popoverBehavior = 'auto' } = options ?? {};
+  const { popoverBehavior = 'manual' } = options ?? {};
```

Comment:
We shouldn't change the default. This can be overridden from the consumer instead